components/function-controller/internal/controllers/serverless/config.go

```diff
@@ -24,7 +24,8 @@ type BuildConfig struct {
 	LimitsMemory         string            `envconfig:"default=1Gi"`
 	LimitsMemoryValue    resource.Quantity `envconfig:"-"`
 	RuntimeConfigMapName string            `envconfig:"default=dockerfile-nodejs-12"`
-	ExecutorImage        string            `envconfig:"default=gcr.io/kaniko-project/executor:v0.19.0"`
+	ExecutorArgs         []string          `envconfig:"default=--insecure;--skip-tls-verify;--skip-unused-stages;--log-format=text;--cache=true"`
```

Comment:
--insecure and --skip-tls-verify shouldn't be set from env

Reply:
Not exactly, this is only valid when our internal docker registry is used